production.Dockerfile

```diff
@@ -113,7 +112,6 @@ ENV PATH=/python-runtime/bin:$PATH \
 COPY manage.py manage.py
 COPY hogvm hogvm/
 COPY posthog posthog/
-COPY ee ee/
```

Comment:
Hi @muscache, unfortunately we can't do this, because this is also the image used by PostHog Cloud. We should bring posthog-foss back online instead. This is in our backlog, but it's hard for me to provide a timeline for this :(

Reply:
My experience with "it's on our backlog" is that it's never going to happen. Of course that's entirely fair since it's essentially a donation. That said, I feel it might be more helpful if you could instead give some idea of what the plan is there so that the community might take it up and submit a PR for it? Then it doesn't cost you anything, and if there's really nobody that cares enough to submit a PR to fix it, well, then they don't want it enough 🙂